.rubocop.yml

```diff
@@ -26,7 +26,7 @@ Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
+  Max: 200
```

Comment:
- [ ] Don't make changes in the rubocop config file

Reply:
ok, thanks